dependencies.gradle

```diff
@@ -89,7 +87,7 @@ ext {
             appCompat          : "androidx.appcompat:appcompat:$appCompatVersion",
             constraintLayout   : "androidx.constraintlayout:constraintlayout:$constraintLayoutVersion",
             coreKtx            : "androidx.core:core-ktx:$coreKtxVersion",
-            window             : "androidx.window:window:$windowVersion",
+            windowAlpha        : "androidx.window:window:$windowAlphaVersion",
```

Comment:
When the window manager will be released, please don't forget to rename again this constant.